tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -182,11 +194,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                     jQueryTrigger: "focus"
                 });
 
-                if (link.hasClass(pager.pagerBar.options.styles.currentPage)) { // assumption that this will not change between now and then!
-                    assertVisibleTips("There shouldn't be any tooltips visible when the currentPage is focused", []);
-                } else {
-                    assertVisibleTips("Only the tooltip for page link " + (idx + 1) + ", in the " + location + " page bar is visible", [linkId]);
-                }
+                
+                assertVisibleTips("There should be tooltip visible when none of the pageLinks are focused", [linkId]);
```

Comment:
I'm a bit confused about what this is testing. Why is there a tooltip if none of the pageLinks are focused? Is something hovered?